### PR TITLE
Fix CSV unicode error

### LINKED ISSUE #2947: Emojis not encoded properly in CSV export
## Issue - Bug

A user requested this one:

> Hi,
>
>When I download the CSV with I can notice some weird characters if the user use emojis
>
> unnamed
>
>Maybe when exporting the emoji should be converted into a code like `:stopwatch:`

- **Estimated effort:** 10.5 points ([see CONTRIBUTING.md](https://github.com/ParabolInc/action/blob/master/CONTRIBUTING.md#points-and-sizes))

### Acceptance Criteria (optional)
Users can:
   - Download a CSV file containing emojis and see them properly encoded, or, see a English textual representation of them

- **Estimated effort:** 10.5 points ([see CONTRIBUTING.md](https://github.com/ParabolInc/action/blob/master/CONTRIBUTING.md#points-and-sizes))

### PR DESCRIPTION
**Pull request by mikennaji**
Users are able to see Emojis on CSV export with Unicode shown properly.

Closes #2947

### CODE REVIEW

File: packages/client/modules/email/components/SummaryEmail/ExportToCSV.tsx [hunk 1]
```diff
@@ -190,7 +190,9 @@ class ExportToCSV extends Component<Props> {
     const {endedAt, team, meetingType} = newMeeting
     const {name: teamName} = team
     const label = meetingType[0].toUpperCase() + meetingType.slice(1)
-    const parser = new Parser()
+    const withBOM = true
+    const opts = {withBOM}
+    const parser = new Parser(opts)
```
Comment (jordanh) [inline packages/client/modules/email/components/SummaryEmail/ExportToCSV.tsx:195]: Can you please compact this to 1 line, not 3?

Comment (mikennaji) [reply packages/client/modules/email/components/SummaryEmail/ExportToCSV.tsx]: Is it okay now?

**Comment by mattkrick** [comment]
couldn't reproduce on macos, but i'm betting apple does some behind the scenes stuff to make it work with bad encoding, happy to merge the `withBOM` fix

**Comment by mikennaji** [comment]
Thank you @mattkrick ! The error occurs on Open Office and Excel.

File: packages/client/modules/email/components/SummaryEmail/ExportToCSV.tsx (hunk 1 above)
Comment (jordanh) [reply packages/client/modules/email/components/SummaryEmail/ExportToCSV.tsx]: Can you make this...

```javascript
const parser = new Parser({ withBOM: true})
```
 
...instead?

**Comment by mattkrick** [comment]
@mikennaji merged! this will be in v5.6.0, should be live in an hour or so

**Pull request merged**